src/noted.cpp

```diff
@@ -0,0 +1,82 @@
+#include <argp.h>
+#include <iostream>
+#include <libnotify/notify.h>
+#include "../include/note.hpp"
+#include <stdlib.h>
+#include <stdio.h>
+#include <errno.h>
+#include <sys/types.h>
+#include <sys/inotify.h>
+#include <limits.h>
+#include <chrono>
+#include <thread>
+#include <ctime>
+#define NOTE_DUE "One of your notes just reached its due time!"
```

Comment:
Why not use a static const?

Reply:
libnotify is a C library, so it requires a `char*`. Anyways, changes this to `const char* NOTE_DUE = ...`